trikotFoundation/src/iosMain/kotlin/com/mirego/trikot/foundation/concurrent/dispatchQueue/IOSSerialDispatchQueue.kt

```diff
@@ -20,12 +22,24 @@ open class IOSSerialDispatchQueue(identifier: String) : TrikotDispatchQueue {
         dispatch_async(
             serialQueue,
             freeze {
-                runQueueTask(block)
+                try {
+                    runQueueTask(block)
```

Comment:
This seems to be repeated 3 times, maybe put in common?

Reply:
Good point. Will do 👍🏻

Reply:
I extracted the `forwardExceptionOnMainThread` block in a new `TrikotDispatchQueue` extension in `nativeMain`. I did not extract the `runQueueTask` in an extension as all dispatch queues already have a method called like that. Only IOSGlobalDispatchQueue and IOSSerialDispatchQueue should use that `try catch`.